cli_reference/basic_cli_operations.adoc

```diff
@@ -10,34 +10,56 @@
 toc::[]
 
 == Overview
-This topic provides information on some general CLI operations and their syntax.
+This topic provides information on some general CLI operations and their syntax,
+and assumes you have gone through the link:get_started_cli.html[basic setup and
+login] for the CLI.
 
-== Common CLI Operations
-The CLI allows interaction with the various objects that are managed by OpenShift. The common `osc` operations are invoked by specifying the following:
+== Common Operations
+The CLI allows interaction with the various objects that are managed by OpenShift. Many common `osc` operations are invoked using the following syntax:
 
-* An _action_
-* The _object type_ to perform the action on
-* The _object name_ or _ID_ of the specified _object type_
+****
+`$ osc _<action>_ _<object_type>_ _<object_name_or_id>_`
+****
+
+This specifies:
 
-In the following example the `osc get` operation returns a complete list of services that are currently defined.
+- An `_<action>_` to perform, such as `get` or `describe`.
+- The `_<object_type>_` to perform the action on, such as `service` or the abbreviated `svc`.
+- The `_<object_name_or_id>_` of the specified `_<object_type>_`.
 
+For example, the `osc get` operation returns a complete list of services that are currently defined:
+
```

Comment:
[Suggestion]
The following example shows the `osc get` operation return a complete list of services that are currently defined.

[Example]
Wrap the example in an example block.